src/Razor/src/Microsoft.AspNetCore.Razor.Language/TagHelperMatchingConventions.cs

```diff
@@ -128,7 +128,9 @@ public static bool SatisfiesAttributes(IEnumerable<KeyValuePair<string, string>>
 
         public static bool CanSatisfyBoundAttribute(string name, BoundAttributeDescriptor descriptor)
         {
-            return SatisfiesBoundAttributeName(name, descriptor) || SatisfiesBoundAttributeIndexer(name, descriptor);
+            return SatisfiesBoundAttributeName(name, descriptor) ||
+                SatisfiesBoundAttributeIndexer(name, descriptor) ||
+                descriptor.BoundAttributeParameters.Any(p => SatisfiesBoundAttributeParameter(name, descriptor, p));
```

Comment:
You can't require parameters so does this addition ever actually matter? Aka, I think `SatisfiesBoundAttributeName` above will return true and this added case will never actually run.

Reply:
`SatisfiesBoundAttributeName` will not return true because `name` in this case might be `bind:event` but it'll expect the `name` to be `bind`. This is why `SatisfiesBoundAttributeParameter` contains a `SatisfiesBoundAttributeName` check as well.

Reply:
I see. 👍